deepinv/unfolded/deep_equilibrium.py

```diff
@@ -128,25 +100,42 @@ def DEQ(
     **kwargs
 ):
     r"""
-    Function instantiating a DEQ algorithm.
-
-    :param str algo_name: name of the algorithm to be used. Should be either `"PGD"`, `"ADMM"`, `"HQS"`, `"PD"` or `"DRS"`.
-    :param dict params_algo: dictionary containing the parameters of the algorithm.
-    :param list trainable_params: list of trainable parameters. Default: `[]`.
-    :param deepinv.optim.data_fidelity data_fidelity: data fidelity term in the optimization problem.
-    :param F_fn: Custom user input cost function. Default: None.
-    :param g_first: whether to perform the step on :math:`g` before that on :math:`f` before or not. Default: False.
+    Function building the appropriate Unfolded architecture.
+
+    :param algo: either name of the algorithm to be used, or an iterator. If an algorithm name (string), should be either `"PGD"`, `"ADMM"`, `"HQS"`, `"CP"` or `"DRS"`.
```

Comment:
Should we call it iterator as in the optim builder?

Reply:
The input type with 'or' ?

Reply:
The input type, i.e., bool

:param str, torch.nn.Module iterator: ....
:param bool g_first: ...